.docker/ci/Dockerfile

```diff
@@ -29,20 +32,55 @@ RUN \
         wget \
         # Required for rosdep command
         sudo \
-        # Preferred build tools
-        python-catkin-tools \
         clang clang-format-3.9 clang-tidy clang-tools \
-        ccache && \
+        ccache \
+        curl gnupg2 lsb-release \
+        # Some dependencies for working with ROS2
```

Comment:
These are not pulled in by the ROS2 base image anymore?

Reply:
Many of these dependencies are already available but many others such as pytest, setuptools, pip, and flake8 are not. 

I have pruned down the list in my latest updates.